src/lambdas/closeZendeskTicket/handler.ts

```diff
@@ -12,9 +12,6 @@ import {
 
 export const handler = async (event: SQSEvent, context: Context) => {
   initialiseLogger(context)
-  logger.info('Handling close zendesk ticket SQS event', {
```

Comment:
Can we log out the ticket id that is being closed instead of deleting?

Reply:
An issue with logging out the ticket id here is that this log ran before validation as carried out on the event, and therefore you can't be sure that the event has a ticket id at this stage.  The next block of code parses the event and  checks for the zendesk ticket ID - if it's not there an error is thrown, if it is there (and all other expected fields are too) then the zendesk ID is appended to the logger and so will appear in logs.